okdownload/src/main/java/com/liulishuo/okdownload/core/file/MultiPointOutputStream.java

```diff
@@ -299,6 +301,15 @@ void inspectStreamState(StreamsState state) {
         boolean isNoMoreStream = true;
         state.newNoMoreStreamBlockList.clear();
 
+        final int totalBlockCount = info.getBlockCount();
+        final int outputStreamCreatedBlockCount = outputStreamCreatedBlockCounter.get();
```

Comment:
the count of `outputStream` will accept to not equal to the `totalBlockCount` when some blocks have been finished before download this time.

Reply:
One another case, when a connection is failed or even not finish connect, it's also accepted `not equal`.

Reply:
updated